.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - name: Check TOML Syntax
+        id: check-toml
+        alias: toml
+      - name: Check YAML Syntax
+        id: check-yaml
+        alias: yaml
+      - name: Fix EOFs
+        id: end-of-file-fixer
+        alias: eof
+      - name: Strip trailing whitespace
+        id: trailing-whitespace
+        alias: whitespace
```

Comment:
I don't think we need an additional `name` tag here, the `id` is quite straightforward and understandable.